show/bgp_common.py

```diff
@@ -0,0 +1,394 @@
+import copy
+import ipaddress
+import json
+
+import utilities_common.multi_asic as multi_asic_util
+from sonic_py_common import multi_asic
+
+# show ip(v6) route helper methods start
+# Helper routines to support print_ip_routes()
```

Comment:
Prefer 'docstring'. They can be used to generate documentation.

Reply:
Sure. I will change it.